src/redux/movies/reducer.test.js

```diff
@@ -0,0 +1,129 @@
+import { moviesReducer } from './reducer';
+import { MoviesActionsTypes } from './types';
+
+describe('moviesReducer', () => {
+  test('should return the initial state', () => {
+    expect(moviesReducer(undefined, {})).toEqual({
```

Comment:
Not an error, but you can use this object as constant and reuse it in your tests:
```js
const initState = {
      movies: [],
      isFetching: false,
      request: '',
      sortParam: 'release_date',
      filterParam: '',
}

  test('should handle GET_MOVIES_START', () => {
    expect(moviesReducer(initialState,
    {
      type: MoviesActionsTypes.GET_MOVIES_START,
    })).toEqual(initialState);
  });

  test('should handle GET_MOVIES_ERROR', () => {
    expect(moviesReducer(initialState, {
      type: MoviesActionsTypes.GET_MOVIES_ERROR,
      error: 'some error',
    })).toEqual({
      ...initialState,
      error: 'some error',
    });
  });

...
```

but this is up to you

Reply:
thanks!